### DISCUSSION TITLE
Running into issues with s1apintegration test

### DISCUSSION THREAD

**Discussion by shalinjshah18** [magma/magma#7267] [Q&A]
Hi, 
I have setup Magma AGW and Orchestrator on Mac machine using dev environment setup instructions.
Once done, I am running all s1ap integration test from Test VM.

At the point where script runs test_attach_service_with_multi_pdns_and_bearers test, it fails.
After failure script halts and does not move forward.
Attached are logs for failure

Need help how to resolve this issue.

[test_attach_service_with_multi_pdns_and_bearers.log](https://github.com/magma/magma/files/6582907/test_attach_service_with_multi_pdns_and_bearers.log)

**Comment by jayshankarnair** [comment]
>Currently magma-dev is running no process on 192.168.60.142:3333. Hence test fails

**Reply by shalinjshah18** [reply]
Ideally what process shall be running on port 3333?

**Reply by jayshankarnair** [reply]
the request is http://192.168.60.142:3333/stats/switches. I am also checking. May be rest api

**Reply by ulaskozat** [reply]
You should not run s1ap tester when orc8r is connected to the AGW. S1ap tester makes configurations to AGW and those configurations will be overwritten by the Orc8r (e.g., subscribers) and services will get restarted.

**Reply by shalinjshah18** [reply]
Hi Ulas,
I ran all s1ap tests on macbook pro machine while Orc8r services were shutdown and no docker containers were running on machine.

After running s1ap tests end up having error as below:

************************* Sending Service request for UE id  1
************************* send SCTP SHUTDOWN
Deleting Ue Context from Traffic Handler
Keys left in Redis (list should be empty)[
 IMSI001010000000001.magma.ipv4,ipv4:mobilityd_ipdesc_record
IMSI001010000000001:MME
IMSI001010000000001:SPGW

nb_enb_connected: 2
nb_ue_attached: 1 
]
Entries in s1ap_imsi_map (should be zero): 0
Entries left in hashtables (should be zero): 5
F
======================================================================
FAIL: Test Mobile Reachability timer expiry handling
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/magma/lte/gateway/python/integ_tests/s1aptests/test_attach_mobile_reachability_timer_expiry.py", line 111, in test_attach_mobile_reachability_timer_expiry
    response.msg_type, s1ap_types.tfwCmd.INT_CTX_SETUP_IND.value
AssertionError: 40 != 53
-------------------- >> begin captured logging << --------------------
root: ERROR: Could not load magmad yml config for mconfig modules
urllib3.connectionpool: DEBUG: Starting new HTTP connection (1): 192.168.60.142:3333
urllib3.connectionpool: DEBUG: http://192.168.60.142:3333 "GET /stats/switches HTTP/1.1" 200 16
root: INFO: Adding subscriber : IMSI001010000000001
root: DEBUG: s1ap response expected, received: 12, 12
root: DEBUG: s1ap message expected, received: 68, 68
root: INFO: Deleting subscriber : IMSI001010000000001
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 1 test in 364.296s

FAILED (failures=1)
Makefile:42: recipe for target 'integ_test' failed
make: *** [integ_test] Error 1
